core/arch/arm/plat-aspeed/conf.mk

```diff
@@ -0,0 +1,25 @@
+PLATFORM_FLAVOR ?= ast2600
+
+ifeq ($(PLATFORM_FLAVOR),ast2600)
+
+include core/arch/arm/cpu/cortex-a7.mk
+
+$(call force,CFG_8250_UART,y)
+$(call force,CFG_ARM32_core,y)
+$(call force,CFG_TEE_CORE_NB_CORE,2)
+$(call force,CFG_TEE_CORE_DEBUG, y)
+$(call force,CFG_TEE_CORE_LOG_LEVEL, 4)
```

Comment:
For these two at least you probably should use `?=`. See https://optee.readthedocs.io/en/latest/building/gits/optee_os.html#configuration-variables and https://optee.readthedocs.io/en/latest/building/gits/optee_os.html#the-force-macro.

Reply:
Yes.
These two are for debugging purposes during my development.
I will remove them from the force-macro so that they can be enabled by demands.

Reply:
Revise as suggested with 4c79cba9a3f7d0e25923a222312627047b46f645